packages/components/src/index.tsx

```diff
@@ -0,0 +1,73 @@
+export * from "./AnimatedPresence";
+export * from "./AnimatedSwitcher";
+export { Autocomplete } from "./Autocomplete";
+export * from "./Avatar";
+export * from "./Banner";
+export * from "./Button";
+export * from "./ButtonDismiss";
+export * from "./Card";
+export * from "./Checkbox";
+export * from "./Chip";
+export { Chips } from "./Chips";
+export * from "./Combobox";
+export * from "./ConfirmationModal";
+export * from "./Content";
+export * from "./Countdown";
+export * from "./DataDump";
+export * from "./DataList";
+export { DataTable, createColumnHelper } from "./DataTable";
```

Comment:
There are some component name collisions if * is exported from the items in this list with only named exports.